kvdb-rocksdb/CHANGELOG.md

```diff
@@ -5,9 +5,17 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
-- use `get_pinned` API to save one allocation for each call to `get()` (See [PR #274](https://github.com/paritytech/parity-common/pull/274) for details)
-- rename `drop_column` to `remove_last_column` (See [PR #274](https://github.com/paritytech/parity-common/pull/274) for details)
-- rename `get_cf` to `cf` (See [PR #274](https://github.com/paritytech/parity-common/pull/274) for details)
+- Use `get_pinned` API to save one allocation for each call to `get()` (See [PR #274](https://github.com/paritytech/parity-common/pull/274) for details)
+- Rename `drop_column` to `remove_last_column` (See [PR #274](https://github.com/paritytech/parity-common/pull/274) for details)
+- Rename `get_cf` to `cf` (See [PR #274](https://github.com/paritytech/parity-common/pull/274) for details)
+- Default column support removed from the API (See [PR #278](https://github.com/paritytech/parity-common/pull/278) for details)
+  - Column argument type changed from `Option<u32>` to `u32`
+  - Migration
+    - Column index `None` -> `0`, `Some(0)` -> `1`, `Some(1)` -> `2`, etc.
```

Comment:
This is not quite correct, see https://github.com/paritytech/parity-common/issues/272#issuecomment-562618192